response_operations_ui/views/collection_exercise.py

```diff
@@ -117,9 +118,6 @@ def view_collection_exercise(short_name, period):
     if sample_controllers.sample_summary_state_check_required(collection_exercise["state"], sample):
         try:
             sample_load_status = sample_controllers.check_if_all_sample_units_present_for_sample_summary(sample["id"])
-            if sample_load_status["areAllSampleUnitsLoaded"]:
```

Comment:
I don't think this has ever/recently ever worked and we recently removed a pre formatting of the sample load time prior to this check which means that now we are passing an unformatted time to the template.